src/comment.js

```diff
@@ -2,6 +2,11 @@ import { details, summary, b, fragment, table, tbody, tr, th } from "./html";
 import { percentage } from "./lcov";
 import { tabulate } from "./tabulate";
 
+const renderEmoji = pdiff => {
+	if (pdiff < 0) return "🔴";
+	if (pdiff > 0) return "🟢";
```

Comment:
Lets use `>= 0` here, so that we always have an emoji and that 0% shows as green :)